interface/wx/listctrl.h

```diff
@@ -1088,6 +1088,13 @@ class wxListCtrl : public wxControl
     */
     void SetImageList(wxImageList* imageList, int which);
 
+    /**
+        Check if the item is visible
```

Comment:
```suggestion
        Check if the item is visible.
```

Reply:
This and the next comment are still relevant.